app/scripts/filters/resources.js

```diff
@@ -955,6 +955,17 @@ angular.module('openshiftConsole')
       return annotationFilter(build, 'jenkinsBuildURL');
     };
   })
+  .filter('jenkinsInputURL', function(jenkinsBuildURLFilter) {
+    return function(build) {
+      // TODO we should have a jenkinsInputURL annotation, waiting on
+      // https://github.com/fabric8io/openshift-jenkins-sync-plugin/issues/94
+      var buildURL = jenkinsBuildURLFilter(build);
+      if (!buildURL) {
+        return null;
+      }
+      return new URI(buildURL).segment("/input/").toString();
```

Comment:
I'm hesitant to assume anything about the Jenkins URL patterns since they could change. I don't see this as a documented API.

https://github.com/jenkinsci/pipeline-stage-view-plugin/tree/master/rest-api

I hope the sync plugin can add this, and this is only a stop-gap.

@bparees FYI